test/functional/p2p_blockfilters.py

```diff
@@ -78,7 +92,7 @@ def run_test(self):
         assert int(self.nodes[0].getnetworkinfo()['localservices'], 16) & NODE_COMPACT_FILTERS != 0
         assert int(self.nodes[1].getnetworkinfo()['localservices'], 16) & NODE_COMPACT_FILTERS == 0
 
-        self.log.info("get cfcheckpt on chain to be re-orged out.")
+        self.log.debug("get cfcheckpt on chain to be re-orged out.")
```

Comment:
pico-nit, if you retouch maybe (a) s/get/Get/ and (b) can remove the EOL full stop per most frequent practice in the functional test logging